### DISCUSSION TITLE
2023 Q2 Technical Manager Monthly Position Proposal

### DISCUSSION THREAD

**Discussion by furoxr** [ringdao-community-guild/discussion#3] [General]
## Introduction:

I am seeking funding to support several critical projects related to communication, blockchain, and community development. These projects are essential for the smooth functioning of various networks and communities, and I am committed to delivering high-quality work to ensure their success. The projects I am working on are as follows:

* Bridger: <https://github.com/darwinia-network/bridger>

* Web3 Grant: <https://github.com/w3f/Grants-Program/pull/1546>

* Darwinia Community DAO(DCDAO) Profiles Repository: <https://github.com/dcdao/profiles>

* DCDAO Snapshot maintenance: <https://snapshot.org/#/darwiniacommunitydao.eth>

* Discord AI Bot with the knowledge of Darwinia Network: <https://github.com/furoxr/discord-ai-bot>

* Slack-Discord messages bridge

  I am responsible for maintaining a bridge that connects Slack and Discord channels, allowing users from different platforms to communicate seamlessly.

## Target of this Quarter

Development and maintenance:

* Bridger: Relayer part of the message bridge of Darwinia<>ETH

  1. Migrate from rust-web3 to ethers-rs since [rust-web3 is no longer maintained](https://github.com/tomusdrw/rust-web3#status)

  2. Handle transaction collision

  3. Keep up with Ethereum and contract updates

  4. Integration tests and maintenance of the indexer

* Web3 Grant: Accountable light client

  Research zero-knowledge proof and its applications. I will cooperate with Aki and Echo to implement the accountable light client verifier, including development, tests, and I will try to write some articles to help people who are interested in following the theory behind the scenes.

* I will maintain and upgrade the slack-discord message bridge bot, and discord ai bot, which include the operation with a cloud server, and related api service.

* I will maintain and upgrade the repository of [DCDAO Profiles](https://github.com/dcdao/profiles) to optimize the automated workflow to promte the application and audit.

* I will try my best to handle other technical problems encountered or discovered during the communication with other community members.

## Budget

The funding I am seeking is $3200 every month(Both USDT and RING are acceptable), which includes the development and maintenance of related services.

My ethereum address is: 0xa6B5EC845C8446A350a50bb58363c241ab7990b5

## Conclusion

I am confident that the proposed projects will have a significant impact on the communication, blockchain, and community development domains. The funding will allow me to enhance my solutions and ensure their ongoing maintenance, which will benefit users and Darwinia Network. I appreciate your consideration of my proposal and look forward to the opportunity to work together.

**Comment by hackfisher** [comment]
> The funding I am seeking is $4000 every month(Both USDT and RING are acceptable), which includes the development and maintenance of related services.

The funding is relative hight than the current expectation($3200) for this position and budget strategy cc @1022ranji, https://github.com/orgs/dcdao/discussions/6, I would suggest adjust and hold to $3200, otherwise I would vote Nay for it in snapshot proposal.

**Comment by furoxr** [comment]
https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x7cd457bd07cdace315189f60e31d1fd217301d8ea5597e461387bb85d5f882fd
